ts/features/profileSettings/store/reducers/index.ts

```diff
@@ -0,0 +1,43 @@
+import { getType } from "typesafe-actions";
+import { PersistConfig, persistReducer } from "redux-persist";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { setShowProfileBanner } from "../actions";
+import { Action } from "../../../../store/actions/types";
+
+export type ProfileSettingsState = {
+  showProfileBanner: boolean;
+};
+
+export const profileSettingsReducerInitialState = {
+  showProfileBanner: true
+};
+
+const profileSettingsReducer = (
+  state: ProfileSettingsState = profileSettingsReducerInitialState,
+  action: Action
+): ProfileSettingsState => {
+  switch (action.type) {
+    case getType(setShowProfileBanner): {
+      return {
+        ...state,
+        showProfileBanner: action.payload
+      };
+    }
+    default:
+      return state;
+  }
+};
+
+const CURRENT_REDUX_PROFILE_SETTINGS_STORE_VERSION = -1;
+
+const persistConfig: PersistConfig = {
+  key: "profileSettings",
+  storage: AsyncStorage,
+  version: CURRENT_REDUX_PROFILE_SETTINGS_STORE_VERSION,
+  whitelist: ["showProfileBanner"]
+};
```

Comment:
What about the persistence of the citizen's choice between logouts? The choice must be remembered or not? And if yes, what about if a different user enter the app? Has this scenario been discussed? What do you think?

Reply:
As far as I know, this issue has not been discussed. Now that you make me think about it, maybe it would be better to address the topic!